cernopendata/modules/theme/assets/semantic-ui/js/records/components/FileTable.js

```diff
@@ -33,13 +33,16 @@ import config from "../config";
 
 import "./FileTable.scss";
 
-export default function FileTable({ items }) {
+export default function FileTable({ items, table_type }) {
   const [openModal, setOpenModal] = useState(false);
   const [openDownloadModal, setOpenDownloadModal] = useState(false);
   const [selectedFile, setSelectedFile] = useState();
 
-  const getFileUri = (fileKey) => `/record/${config.pidValue}/files/${fileKey}`;
-
+  const getFileUri = (table_type, fileKey) => {
+    var  url="/record/"+config.pidValue+"/"+table_type +"/" + fileKey;
```

Comment:
you can keep the ``(string literals) for readability

Reply:
done